src/+energy/calculateBoundaryConditions.m

```diff
@@ -0,0 +1,43 @@
+function [RHS, EnergyMatrices] = calculateBoundaryConditions(BoundaryCondition, EnergyMatrices, HBoundaryFlux, ForcingData, ...
+                                                             SoilVariables, Precip, EVAP, Delt_t, r_a_SOIL, Rn_SOIL, RHS, L, KT)
+    %{
+        Determine the boundary condition for solving the energy equation.
```

Comment:
This function is missing a reference. Is one available (e.g. the STEMMUS technical manual) or is it undocumented?

Reply:
Looking at issue #100, I assume the reference is STEMMUS Technical Notes. Added.